src/lib/rest/RestService.cpp

```diff
@@ -443,7 +444,7 @@ static bool compErrorDetect
 * we need to know if it's v1 or v2.
 *
 * This is very easy, we just look at the first component of the URI PATH, and
-* if it is "v2" or "V2", then case-sensitive checks are used.
+* if it is 2 or 2, then case-sensitive checks are used.
```

Comment:
This string must change, but it must also be thoroughly read to understand what has happened here ...